heed/src/env.rs

```diff
@@ -260,6 +261,53 @@ pub enum CompactionOption {
 }
 
 impl Env {
+    /// Return the size used by all the databases in the environment.
+    pub fn size(&self) -> Result<usize> {
+        let compute_size = |stat: lmdb_sys::MDB_stat| {
```

Comment:
Not sure that you need to specify the type here. Rust will guess it by itself.

```suggestion
        let compute_size = |stat| {
```

Reply:
I don’t know why but _no_
<img width="507" alt="image" src="https://user-images.githubusercontent.com/7032172/183871474-248af458-b1da-4d75-a928-7f533b2b313f.png">